packages/client/src/api/forumApi/forumApi.ts

```diff
@@ -0,0 +1,26 @@
+import { forumAxiosInstance } from '@api/baseAxiosInstance';
+
+const getAllForums = () => {
+  return forumAxiosInstance.get('/forums');
+};
+
+const getAllTopics = (id: string) => {
```

Comment:
некоторые IDE показывают название аргумента, поэтому для простоты понимания будет круче наименовать 
```suggestion
const getAllTopics = (forumId: string) => {
```

Reply:
Спасибо за совет!